src/objects/Trigger_Handler__c.object

```diff
@@ -89,6 +89,16 @@
         <unique>false</unique>
     </fields>
     <fields>
+        <fullName>Exclude_For_These_Usernames__c</fullName>
```

Comment:
I agree that the Developer Name and the Label should match (so it's easier to identify regardless if you're looking at one or the other). Could we go with something simpler like Developer Name: Users_to_Exclude and Label: Users to Exclude?